pkg/reconcile/runner.go

```diff
@@ -80,6 +97,15 @@ func (r *runner) RunReconciler(ctx context.Context, reconciler Reconciler, predi
 	if err != nil {
 		return err
 	}
+
+	if r.options.Verifier != nil {
+		if resourceRegistered, err := r.options.Verifier.VerifyServerResource(r.cluster, r.mgr.GetConfig(), gvk); err != nil {
+			return err
+		} else if !resourceRegistered {
+			return nil
```

Comment:
Should we log here to explain what happened, or maybe return a different error, because it could be confusing for nothing to happen

Reply:
if the user sets the verify option to `Warn` an warning will be logged